test/e2e/usersignup_test.go

```diff
@@ -136,6 +136,33 @@ func (s *userSignupIntegrationTest) TestAutomaticApproval() {
 	})
 }
 
+func (s *userSignupIntegrationTest) TestNotificationCreatedOKWhenUserProvisioned() {
+	// given
+	s.Host().UpdateToolchainConfig(testconfig.AutomaticApproval().Enabled(true))
+
+	// when
+	userSignup, _ := NewSignupRequest(s.T(), s.Awaitilities).
+		Username("bartsimpson").
+		Email("bartsimpson@redhat.com").
+		RequireConditions(ConditionSet(Default(), ApprovedAutomatically())...).
+		Execute().Resources()
+
+	// then
+	userSignup, err := s.Host().WaitForUserSignup(userSignup.Name,
+		wait.UntilUserSignupHasConditions(ConditionSet(Default(), ApprovedAutomatically())...),
```

Comment:
isn't the presence of the conditions already ensured by `RequireConditions(ConditionSet(Default(), ApprovedAutomatically())...).` ?